sysbuild/CMakeLists.txt

```diff
@@ -86,6 +90,20 @@ function(${SYSBUILD_CURRENT_MODULE_NAME}_pre_cmake)
         )
     endif()
   endif()
+
+  if(SB_CONFIG_BT_FAST_PAIR)
+    set_config_bool(${DEFAULT_IMAGE} CONFIG_BT_FAST_PAIR y)
+
+    if(DEFINED FP_MODEL_ID AND DEFINED FP_ANTI_SPOOFING_KEY)
+      include_fast_pair_hex()
+      set(FP_DATA_PRESENT "y" CACHE INTERNAL "Fast pair provision data provided" FORCE)
```

Comment:
`Fast Pair provisioning data provided` to be consistent with other places

Reply:
Fixed